critiquebrainz/db/revision.py

```diff
@@ -148,23 +152,34 @@ def get_revision_number(review_id, revision_id):
     return rev_num
 
 
-def create(review_id, text):
+def create(review_id, text, rating):
```

Comment:
What if I have a review without a rating? I shouldn't need to worry about how revisions are stored when I use this function. This function might be used in other places in the future, so it is better to do the check here as well.

Arguments can always be modified however necessary before they are inserted into the database.